src/js.rs

```diff
@@ -148,6 +148,12 @@ extern "C" {
     #[wasm_bindgen(method)]
     pub fn find(this: &Array, predicate: &mut FnMut(JsValue, u32, Array) -> bool) -> JsValue;
 
+    /// The forEach() method executes a provided function once for each array element.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
+    #[wasm_bindgen(method, js_name = forEach)]
+    pub fn for_each(this: &Array, callback: &Function) -> JsValue;
```

Comment:
I think the `&Function` may want to perhaps be a variant of `&mut FnMut(...)` to invoke the closure in Rust?

Reply:
Yes, we want to use `&mut FnMut(...)` for the callback.

@FrankHassanabad, see the bindings for `find` or `filter` for examples.